diff --git a/app/wordpress/wp-activate.php b/app/wordpress/wp-activate.php
index 6aae1ca..6b38289 100644
--- a/app/wordpress/wp-activate.php
+++ b/app/wordpress/wp-activate.php
@@ -103,7 +103,7 @@ get_header( 'wp-activate' );
 					printf(
 						/* translators: 1: login URL, 2: username, 3: user email, 4: lost password URL */
 						__( 'Your account has been activated. You may now <a href="%1$s">log in</a> to the site using your chosen username of &#8220;%2$s&#8221;. Please check your email inbox at %3$s for your password and login instructions. If you do not receive an email, please check your junk or spam folder. If you still do not receive an email within an hour, you can <a href="%4$s">reset your password</a>.' ),
-						network_site_url( 'wp-login.php', 'login' ),
+						site_url( 'wp-login.php', 'login' ),
 						$signup->user_login,
 						$signup->user_email,
 						wp_lostpassword_url()
