diff --git a/app/moodle/admin/renderer.php b/app/moodle/admin/renderer.php
index 4ee8e3d..9824ad1 100755
--- a/app/moodle/admin/renderer.php
+++ b/app/moodle/admin/renderer.php
@@ -802,22 +802,22 @@ class core_admin_renderer extends plugin_renderer_base {
      */
     protected function registration_warning($registered) {
 
-        if (!$registered) {
-
-            if (has_capability('moodle/site:config', context_system::instance())) {
-                $registerbutton = $this->single_button(new moodle_url('/admin/registration/register.php',
-                    array('huburl' =>  HUB_MOODLEORGHUBURL, 'hubname' => 'Moodle.net')),
-                    get_string('register', 'admin'));
-                $str = 'registrationwarning';
-            } else {
-                $registerbutton = '';
-                $str = 'registrationwarningcontactadmin';
-            }
-
-            return $this->warning( get_string($str, 'admin')
-                    . '&nbsp;' . $this->help_icon('registration', 'admin') . $registerbutton ,
-                'error alert alert-danger');
-        }
+        // if (!$registered) {
+        //
+        //     if (has_capability('moodle/site:config', context_system::instance())) {
+        //         $registerbutton = $this->single_button(new moodle_url('/admin/registration/register.php',
+        //             array('huburl' =>  HUB_MOODLEORGHUBURL, 'hubname' => 'Moodle.net')),
+        //             get_string('register', 'admin'));
+        //         $str = 'registrationwarning';
+        //     } else {
+        //         $registerbutton = '';
+        //         $str = 'registrationwarningcontactadmin';
+        //     }
+        //
+        //     return $this->warning( get_string($str, 'admin')
+        //             . '&nbsp;' . $this->help_icon('registration', 'admin') . $registerbutton ,
+        //         'error alert alert-danger');
+        // }
 
         return '';
     }
